src/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -266,6 +264,7 @@ public void ResolveFieldFail(int token)
 
         [Theory]
         [MemberData(nameof(BadResolveStrings))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Module.Resolve apis not supported on UapAot.")]
```

Comment:
Where you're indicating API on reflection is never going to be implemented on AOT, I'm assuming you've got consensus around that. You and @DnlHarvey own reflection so it's your call in general...